tests/testInitializeSolutionUtils.cpp

```diff
@@ -113,7 +113,7 @@ TEST(InitializeSolutionUtils, InitializePosesAndJoints) {
   auto l1 = robot.link("l1");
   auto l2 = robot.link("l2");
 
-  Pose3 wTb_i = l2->wTcom();
+  Pose3 wTb_i = l2->bTcom();
```

Comment:
Hmmm, what is wTb_i ? Does the code make sense if renamed to `bMcom_2` ? Similar questions below.

Reply:
I think the i stands for "initial" here. This test initializes poses in time using interpolation before optimization (I think). I suggest `bMcom2_init` and `bTcom2_dt` ( bTcom because I think these don't represent rest matrices any more after init?).

Reply:
_i should be reserved for link index. If it's "init" spell out init. But check it's true

Reply:
wTb and wTl are still there in this test, still need to fix that